cardano-coin-selection.cabal

```diff
@@ -86,12 +109,12 @@ test-suite unit
     , text
     , transformers
     , vector
-  build-tools:
-      hspec-discover
+  build-tool-depends:
+      hspec-discover:hspec-discover
```

Comment:
Needed to comply with the stricter syntax rules of version 2 of the `cabal` file format.